web-api/workflow-terraform/migration/main/lambdas/migrations/0007-update-corporate-disclosure-document.ts

```diff
@@ -9,7 +9,7 @@ const isCase = item => {
 };
 
 export const migrateItems = items => {
-  const itemsAfter = [];
+  const itemsAfter: any[] = [];
```

Comment:
Do we really not know what type to use here? Wouldn't something like a generic entity or object be better than `any`?